app/src/main/java/ar/com/wolox/android/example/ui/home/news/NewsFragment.kt

```diff
@@ -10,21 +11,43 @@ import javax.inject.Inject
 
 class NewsFragment @Inject constructor() : WolmoFragment<NewsPresenter>(), NewsView {
 
+    private lateinit var newsAdapter: NewsAdapter
     override fun layout(): Int = R.layout.fragment_news
 
     override fun init() {
-        vNewsRecyclerView.layoutManager = LinearLayoutManager(activity)
+        newsAdapter = NewsAdapter(requireContext(), emptyList())
+        vNewsRecyclerView.apply {
```

Comment:
this should be run instead of apply, because you are not returning nothing. Also, the `addOnScrollListener` should be inside this run